tests/ui/manual_is_ascii_check.rs

```diff
@@ -15,6 +15,14 @@ fn main() {
     assert!(matches!('x', 'A'..='Z' | 'a'..='z'));
 
     assert!(matches!('x', 'A'..='Z' | 'a'..='z' | '_'));
+
+    assert!((b'0'..=b'9').contains(&b'0'));
+    assert!((b'a'..=b'z').contains(&b'a'));
+    assert!((b'A'..=b'Z').contains(&b'A'));
+
+    assert!(('0'..='9').contains(&'0'));
+    assert!(('a'..='z').contains(&'a'));
+    assert!(('A'..='Z').contains(&'A'));
```

Comment:
We should have some tests, that handle the non reference case. Something like;

```rs
let cool_letter = &'g';
('0'..='9').contains(cool_letter);
('a'..='z').contains(cool_letter);
('A'..='Z').contains(cool_letter);
```

You also don't need to wrap these into `assert!()` macros either

Reply:
Sorry, I dont see the difference between `('0'..='9').contains(&'0')` and `('0'..='9').contains(cool_letter)`, `cool_letter` dont looks like a non reference, is this a typo?

Reply:
The difference is, that `cool_letter` is already a reference and therefore doesn't need the `&` when it's used as an argument. The implementation handles this fine, I'd just like a test for it, to ensure that it'll be handled even after some changes to the code :)

Reply:
Okay, I'll add it